picard/ui/mainwindow.py

```diff
@@ -84,6 +83,15 @@
 )
 
 
+try:
+    from PyQt5 import QtMultimedia
+except Exception as e:
```

Comment:
I think this should still check for `ImportError` as discussed in #1203 ;)

Reply:
Oh, I missed this one, fixed in 0e478697299baac0cad8ba6cb1ff869010b9a117